### PR TITLE
Add Skype workload

### PR DESCRIPTION
**Pull request by johnrchrdsn**
- Add Skype workload
- Add ApkWorkload extension support to launch non-default packages names.

### CODE REVIEW

File: wlauto/common/android/workload.py [hunk 1]
```diff
@@ -603,7 +616,7 @@ class GameWorkload(ApkWorkload, ReventWorkload):
                   after setup and run"""),
         Parameter('assets_push_timeout', kind=int, default=500,
                   description='Timeout used during deployment of the assets package (if there is one).'),
-        Parameter('clear_data_on_reset', kind=bool, default=True,
+        Parameter('clear_data_on_reset', kind=bool, default=True, override=True,
```
Comment (setrofim) [inline wlauto/common/android/workload.py]: Since this override does not change anything, it should be removed.

File: wlauto/common/android/workload.py [hunk 2]
```diff
@@ -181,6 +187,11 @@ class ApkWorkload(Workload):
                   If ``True``, workload will check that the APK matches the target
                   device ABI, otherwise any APK found will be used.
                   '''),
+        Parameter('clear_data_on_reset', kind=bool, default=True,
```
Comment (ep1cman) [inline wlauto/common/android/workload.py]: Why was this added? It doesn't seem to be used by your workload

File: wlauto/workloads/skype/__init__.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+import time
+from wlauto import AndroidUxPerfWorkload, Parameter
+
+
+class Skype(AndroidUxPerfWorkload):
+
+    name = 'skype'
+    description = '''
+    A workload to perform standard productivity tasks within Skype.  The workload logs in to the
```
Comment (ep1cman) [inline wlauto/workloads/skype/__init__.py]: similarly to the old `skypevideo` workload this should have instructions covering the fact that a skype acocunt needs to be setup. For voice you can just use the skype echo service but for video a second skype account that auto accepts calls is required. 

In light of this I think the workload should default to voice not video too and default the contact_name to `Echo / Sound Test Service`

**Comment by ep1cman** [comment]
This PR should replace the old `skypevideo` workload
